### DISCUSSION TITLE
Why are all code examples written 2x in docs?

### DISCUSSION THREAD

**Discussion by maheshbansod** [vitest-dev/vitest#2235] [General]
This was so confusing. I've been reading this: [Features | Guide | Vitest](https://vitest.dev/guide/features.html)
And all the code examples are written twice in every code block on this page. I tried hard to look for differences but the code seems to be just repeated, then I checked the source of the markdown file [vitest/features.md at main · vitest-dev/vitest](https://github.com/vitest-dev/vitest/blob/main/docs/guide/features.md) and it's only once here.

**Comment by sheremet-va** [comment]
Should be fixed now
